doc/source/conf.py

```diff
@@ -0,0 +1,185 @@
+# -*- coding: utf-8 -*-
+#
+# clmdoc documentation build configuration file, created by
+# sphinx-quickstart on Thu Feb 23 17:14:30 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+# Note that we need a specific version of sphinx_rtd_theme. This can be obtained with:
+# pip install git+https://github.com/esmci/sphinx_rtd_theme.git@version-dropdown-with-fixes
+import sphinx_rtd_theme
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.intersphinx',
+    'sphinx.ext.autodoc',
+    'sphinx.ext.todo',
+    'sphinx.ext.coverage',
+    'sphinx.ext.imgmath',
+    'sphinx.ext.githubpages']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix(es) of source filenames.
+# You can specify multiple suffix as a list of string:
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'ctsm'
+copyright = u'2020, UCAR'
+author = u''
```

Comment:
I think author should probably stay as

"Erik Kluzek, Bill Sacks, Ben Andre, Alice Bertini"

since we are the ones that worked with the bulk of the content. Or it could set set to 

"CTSM software team"

Reply:
I disagree with this. I removed the authorship information because I felt it did not give credit to the appropriate people. There are many people who should get credit, as carefully laid out in https://escomp.github.io/ctsm-docs/versions/master/html/tech_note/Introduction/CLM50_Tech_Note_Introduction.html for the tech note. Furthermore, I'm not sure whether this authorship information actually gets used anywhere (it may, but if it does, I didn't see how at a glance), so it didn't feel worth putting a lot of effort into coming up with an accurate authorship list (especially given that this would be hard to maintain accurately).